packages/shylu/shylu_node/tacho/src/impl/Tacho_Herk.hpp

```diff
@@ -31,10 +31,10 @@ struct HerkAlgorithm {
 };
 
 struct HerkAlgorithm_Team {
-#if defined(KOKKOS_ENABLE_OPENMP)
-  using type = ActiveHostAlgorithm<runsWithOMP()>::type;
-#else
+#if defined(KOKKOS_ENABLE_CUDA) || defined(KOKKOS_ENABLE_HIP)
   using type = ActiveAlgorithm<runsOnCudaOrHIP()>::type;
+#else
+  using type = ActiveHostAlgorithm<runsWithOMP()>::type;
```

Comment:
@iyamazaki does this also work when Serial backend is enabled as well? If so, maybe the algorithm could be renamed similar to the device version above like `runsWithOMPOrSerial`, otherwise is there a location in the code that will error/abort/throw if attempting to use this with the Serial backend?

Reply:
Thank you, @ndellingwood. These team-algorithm types are currently used only for non serial backends (OpenMP, Cuda, Hip).